content/governance.md

```diff
@@ -0,0 +1,175 @@
+---
+title: Governance
+layout: page
+
+---
+
+# Governance
+
+This document describes the rules and governance of the project. It is meant to be followed by all the developers of the project and the Prometheus community. Common terminology used in this governance document are listed below:
+
+* **Team members**: Any members of the private [prometheus-team][team] Google group.
+
+* **Maintainers**: Maintainers lead an individual project or parts thereof ([MAINTAINERS.md][maintainers.md]).
+
+* **Projects**: Any repository in the Prometheus [GitHub organization][gh].
+
+* **The Prometheus Project**: The sum of all activities performed under this governance, concerning one or more Repositories or the community.
+
+## Values
+
+The Prometheus developers and community are expected to follow the values defined in the [CNCF charter][charter], including the [CNCF Code of Conduct][coc]. Furthermore, the Prometheus community strives for kindness, giving feedback effectively, and building a welcoming environment. The Prometheus developer community strives to operate by rough consensus for the majority of decisions and only resort to conflict resolution through a motion if consensus cannot be reached.
```

Comment:
Language nits: This contains the terms "Prometheus developers", "Prometheus community", and "Prometheus developer community". I'd ditch the latter and just use "Prometheus developers" instead, to not confuse the reader. Also, there is "strives" and then "resort" for the same subject (is it meant as singular or plural)? Finally, repeating "strives" reads a bit awkward, and "majority" is kind of redundant, given the rest of the wording.

Suggestion: Change the last sentence to: "Prometheus developers generally decide by consensus and only resort to conflict resolution by a majority vote if consensus cannot be reached."

Reply:
done